### PR TITLE
databases/week2

### PR DESCRIPTION
**Pull request by Rasim-Deniz**

### CODE REVIEW

File: databases/week2/hyf_homework_databases_lesson2.sql [hunk 1]
```diff
@@ -0,0 +1,257 @@
+-- Part 1: Working with tasks
+USE hyf_lesson1;
+
+insert into task (title, description, created, updated, due_date, status_id, user_id) values ('Do the homework', 'Hyf databases lesson2', NOW(),  NOW(), null, 1, 3);
```
Comment (martamatos) [inline databases/week2/hyf_homework_databases_lesson2.sql]: Just a better way of formatting this query:
```
insert into task (title, description, created, updated, due_date, status_id, user_id) 
    values ('Do the homework', 'Hyf databases lesson2', NOW(),  NOW(), null, 1, 3);
```

File: databases/week2/hyf_homework_databases_lesson2.sql [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+  CONSTRAINT `fk_doctor` FOREIGN KEY (`doctor_id`) REFERENCES `doctor` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `diagnosis` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `name` varchar(255) NOT NULL,
+  `details` text NULL DEFAULT NULL,
+  `diagnosis_date` DATETIME NOT NULL,
+  `patient_id` int(10) unsigned NOT NULL,
+  CONSTRAINT `fk_patient` FOREIGN KEY (`patient_id`) REFERENCES `patient` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
```
Comment (martamatos) [inline databases/week2/hyf_homework_databases_lesson2.sql:116]: One diagnosis can only be given to one patient?
I guess it depends how we interpret the diagnosis :)

File: databases/week2/hyf_homework_databases_lesson2.sql [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+
+SELECT *
+FROM task
+JOIN user_task ON
+task.id=user_task.task_id
+JOIN user ON
+user_task.user_id=user.id
+WHERE user.name LIKE "Maryrose%" AND month(created) = 09;
+
+SELECT monthname(created) AS month, COUNT(*) AS count
```
Comment (martamatos) [inline databases/week2/hyf_homework_databases_lesson2.sql]: This query gives me an error. Can you fix it please?

Also you only needed one table here ;)

**Review by martamatos — CHANGES_REQUESTED** [review]
Good job overall! I just need you to fix one exercise :)

File: databases/week2/hyf_homework_databases_lesson2.sql (hunk 2 above)
Comment (Rasim-Deniz) [reply databases/week2/hyf_homework_databases_lesson2.sql:116]: True, I didn't consider that part.

File: databases/week2/hyf_homework_databases_lesson2.sql (hunk 3 above)
Comment (Rasim-Deniz) [reply databases/week2/hyf_homework_databases_lesson2.sql]: I tried to fix, is it true now?

**Pull request merged**